### PR TITLE
Added clarification labels to Static Rigid Body component

### PR DESCRIPTION
**Pull request by moraaar**
Signed-off-by: moraaar <moraaar@amazon.com>

image

### CODE REVIEW

File: Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp [hunk 1]
```diff
@@ -22,14 +22,23 @@ namespace PhysX
 
             if (auto* editContext = serializeContext->GetEditContext())
             {
+                constexpr const char* toolTip = "The entity behaves as a non-movable rigid object in PhysX.";
```
Comment (hultonha) [inline Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp:25]: suggestion: (tagging @bhanuja-s for visibility) Should this maybe be `entity` not `object`? Or should be stick with convention and use `body`? I'm not 100% `object` fits here

**Review by hultonha — COMMENTED** [review]
I really like the idea just have a suggestion about word choice

File: Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp (hunk 1 above)
Comment (moraaar) [reply Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp]: The text is based on the same text from the dynamic rigid body component's tooltip: "The entity behaves as a movable rigid **object** in PhysX."

I'm happy to change it in both cases from `object` to `body`.

Comment (bhanuja-s) [reply Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp]: I would assume they're interchangeable, but I see what you mean @hultonha. I like `body` but can also propose the question to @chanmosq.

Comment (moraaar) [reply Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp]: I'll switch `object` to `body` in both for now to move on.

Comment (moraaar) [reply Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp]: done

Comment (chanmosq) [reply Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp]: I would agree it fits in this context - "body" is a common term with PhysX

Comment (moraaar) [reply Gems/PhysX/Code/Source/EditorStaticRigidBodyComponent.cpp]: Thanks for the confirmation @chanmosq

**Review by bhanuja-s — APPROVED** [review]
Approved!

**Review by greerdv — APPROVED** [review]

**Review by hultonha — APPROVED** [review]

**Pull request merged**